tests/actions.go

```diff
@@ -1212,23 +1210,23 @@ func (info *TidbClusterInfo) QueryCount() (int, error) {
 	return 0, fmt.Errorf("can not find count of ")
 }
 
-func (oa *operatorActions) CreateSecret(info *TidbClusterInfo) error {
-	initSecretName := "set-secret"
-	backupSecretName := "backup-secret"
+func (oa *operatorActions) CreateSecret(info *TidbClusterInfo) (string, string, error) {
+	initSecretName := fmt.Sprintf("%s-set-secret", info.ClusterName)
```

Comment:
the field should be set when `info` created

Reply:
done